### DISCUSSION TITLE
Birthday in calendar

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16405] [Feature improvement]
The Contact module has a `birthday` field. It would be great if the birthdays would be displayed when viewing the calendar. Alternatively, there could be a cron job that adds a calendar entry for each birthday to come in the next 12 months (and removes expired birthdays from the calendar).

**Comment by rskrzypczak** [comment]
It seems to me redundant. 
Currently you can create a widget on the dashboard that will display contacts who have birthday today.
or
You can also create a workflow action that creates a calendar entry or a notification to users informing that a given contact has a birthday today.
